conversions/temperature_conversions.py

```diff
@@ -1,9 +1,11 @@
 """ Convert between different units of temperature """
 
 
-def celsius_to_fahrenheit(celsius: float) -> float:
+def celsius_to_fahrenheit(celsius: float, digits: int = 2) -> float:
```

Comment:
The `round()` parameter is `ndigits`.

Reply:
Please also add a test or two to prove that `ndigits` is respected.

Reply:
fixed

Reply:
tests coming in a new PR